lib/ui/view/channel/new_channel/new_channel_viewmodel.dart

```diff
@@ -1,6 +1,42 @@
+import 'package:flutter/material.dart';
```

Comment:
Nothing should introduce material or cupertino import in the viewmodel, Kindly make use of FormViewModel and read about it, to use the textEditingController there.

Reply:
@isaacadariku the requested changes have been made